lib/Resolver/ClassResolver.php

```diff
@@ -15,20 +14,22 @@
  *  @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-namespace Pimcore\Model\Document\TypeDefinition\Loader;
 
-use Pimcore\Loader\ImplementationLoader\PrefixLoader as BasePrefixLoader;
+namespace Pimcore\Resolver;
```

Comment:
i see that the file was renamed and would take the history of the PrefixLoader along, not sure if is worth to change this and avoid that, as these files are not related to each other

![image](https://user-images.githubusercontent.com/6014195/224019554-f805d4b9-e427-470a-a341-ca565e09aeae.png)

Reply:
I tried to resolve this serveral times, because I noticed this too, but it would always take the history, even tough I created the file from scratch. :/

Reply:
Ah ok, then nevermind, if you already tried, doesn't seem worth to spend more time into it.
Probably will auto-solve once squashed before merging into the main branch